lighthouse-cli/cli-flags.ts

```diff
@@ -72,6 +73,7 @@ export function getFlags() {
         'disable-device-emulation': 'Disable Nexus 5X emulation',
         'disable-cpu-throttling': 'Disable CPU throttling',
         'disable-network-throttling': 'Disable network throttling',
+        'remote-device': 'Enable remote device emulation',
```

Comment:
how about we describe exactly what it does "Disables CPU/Network throttling, emulation, and defaults the protocol port to 9222`?